docs/wiki/[Examples]-Assign-a-Built-in-Policy.md

```diff
@@ -0,0 +1,191 @@
+        "scope": "${current_scope_resource_id}",
+        "enforcementMode": null
+    },
+    "location": "${default_location}",
+    "identity": {
+        "type": "SystemAssigned"
+    }
+}
+```
+
+## Assign the `Deploy default Microsoft IaaSAntimalware extension for Windows Server` Policy at the `es_root` Management Group
+
+You now need to assign the `Deploy default Microsoft IaaSAntimalware extension for Windows Server` policy and in this example, we will assign it at `es_root`. If you don't already have an `archetype_extension_es_root.tmpl.json` file within your custom `/lib` directory, create one and copy the below code in to the file.
+
+```json
+{
+  "extend_es_root": {
+    "policy_assignments": ["Deploy-Antimalware-Ext"],
+    "policy_definitions": [],
+    "policy_set_definitions": [],
+    "role_definitions": [],
+    "archetype_config": {
+      "access_control": {
+      }
+    }
+  }
+}
+```
+
+You should now kick-off your Terraform workflow (init, plan, apply) to apply the new configuration. This can be done either locally or through a pipeline. When your workflow has finished, the `Deploy default Microsoft IaaSAntimalware extension for Windows Server` policy will be assigned at `es_root`.
+
+## Assign the `NIST SP 800-53 Rev. 5` Policy Set at the `es_root` Management Group
+
+You now need to assign the `NIST SP 800-53 Rev. 5` policy set and in this example, we will assign it at `es_root`. Copy the below code in to your `archetype_extension_es_root.tmpl.json` file and save it.
+
+```json
+{
+  "extend_es_root": {
+    "policy_assignments": ["Deploy-Antimalware-Ext", "NIST-SP-800-53-Rev5"],
+    "policy_definitions": [],
+    "policy_set_definitions": [],
+    "role_definitions": [],
+    "archetype_config": {
+      "access_control": {
+      }
+    }
+  }
+}
+```
+
+You should now kick-off your Terraform workflow (init, plan, apply) to apply the new configuration. This can be done either locally or through a pipeline. When your workflow has finished, the `NIST SP 800-53 Rev. 5` policy set will be assigned at `es_root`.
+
+## Assign the `Not allowed resource types` Policy at the `Landing Zones` Management Group
+
+In this example, we will assign it at the `Landing Zones` Management Group. To do this, either update your existing `archetype_extension_es_landing_zones.tmpl.json` file or create one and copy the below code in to it and save.
+
+```json
+{
+  "extend_es_landing_zones": {
+    "policy_assignments": ["Not allowed resource types"],
+    "policy_definitions": [],
+    "policy_set_definitions": [],
+    "role_definitions": [],
+    "archetype_config": {
+      "access_control": {
+      }
+    }
+  }
+}
+```
+
+You should now kick-off your Terraform workflow once again to apply the updated configuration. This can be done either locally or through a pipeline. When your workflow has finished, the `Not allowed resource types` policy will be assigned at the `Landing Zones` Management Group.
+
+```hcl
```

Comment:
remove this as not needed

Reply:
Removed the HCL code for 'terraform apply'